### PR TITLE
Bug 1232177 - handle mutually exclusive flags on oc process

### PR DESCRIPTION
**Pull request by fabianofranz**
@bparees @mfojtik please review, @smarterclayton fyi.

### CODE REVIEW

**Comment by bparees** [comment]
lgtm.

**Comment by smarterclayton** [comment]
This is nice to have but not stop ship.

**Comment by fabianofranz** [comment]
Yep, I already set the BZ to "low".

File: pkg/cmd/cli/cmd/process.go [hunk 1]
```diff
@@ -80,6 +80,15 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 		return kcmdutil.UsageError(cmd, "Must pass a filename or name of stored template")
 	}
 
+	if kcmdutil.GetFlagBool(cmd, "parameters") {
+		mutuallyExclusive := []string{"value", "labels", "output", "output-version", "raw", "template"}
+		for _, flag := range mutuallyExclusive {
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/process.go]: Inline the constant into the range statement ( `range []string{...}` ) for clarity

Comment (fabianofranz) [inline pkg/cmd/cli/cmd/process.go]: @smarterclayton, fixed.

**Comment by smarterclayton** [comment]
LGTM [merge]

**Comment by fabianofranz** [comment]
re[merge]

**Comment by fabianofranz** [comment]
[merge]

**Comment by fabianofranz** [comment]
[ḿerge]

**Pull request merged**